soc/xtensa/intel_adsp/ace_v1x/include/adsp_shim.h

```diff
@@ -25,102 +25,54 @@ struct cavs_shim {
 	uint32_t dspwct1c_lo;
 	uint32_t dspwct1c_hi;
 	uint32_t _unused2[14];
-	uint32_t clkctl;
-	uint32_t clksts;
-	uint32_t hspgctl; /* cAVS 1.5, see cavs_l2lm for 1.8+ */
-	uint32_t lspgctl; /* cAVS 1.5, see cavs_l2lm for 1.8+ */
-	uint32_t hsrmctl; /* cAVS 1.5, see cavs_l2lm for 1.8+ */
-	uint32_t lsrmctl; /* cAVS 1.5, see cavs_l2lm for 1.8+ */
+	uint32_t clkctl;   /* Offset: 0x78 */
+	uint32_t clksts;   /* Offset: 0x7C */
+	uint32_t _unused3[4];
```

Comment:
Wouldn't be better just keep the fields and add a comment saying that are not being currently used ?

Reply:
This is just a source for bugs (and we have a few of those). Code will just work but you will be calling into unused registers that will have no effect.